### PR TITLE
Added plasma glass into cargo

### PR DESCRIPTION
**Pull request by SethLafuente**
## About the PR 
Added plasma glass into cargo and changed the plasma glass recipe into 2 plasma glass instead of 2 plasma but nobody cares about that players don't even craft plasma glass 
**Screenshots**

image

**Changelog**

:cl: Seth
- add: Added plasma glass crates into cargo

### CODE REVIEW

**Comment by Zumorica** [comment]
So why does cargo need to have plasma glass sheets? We should make it so people craft those instead using plasma and glass sheets.

**Comment by SethLafuente** [comment]
> So why does cargo need to have plasma glass sheets? We should make it so people craft those instead using plasma and glass sheets.

I mean I could change into plasma but isn't that stuff suppose to be more valuable or something also sheets fits kind of better

File: Resources/Prototypes/Recipes/Construction/Graphs/window.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
               doAfter: 2
             - material: Glass
               amount: 2
-            - material: Plasma
+            - material: PlasmaGlass
```
Comment (Zumorica) [inline Resources/Prototypes/Recipes/Construction/Graphs/window.yml:14]: Two things. First one, I really don't like this change.
Second one, why would you change Plasma to PlasmaGlass and leave Glass unchanged?

File: Resources/Prototypes/Catalog/Cargo/cargo_materials.yml [hunk 2]
```diff
@@ -45,3 +45,15 @@
   cost: 6000
   category: Materials
   group: market
+
+- type: cargoProduct
+  name: "crate of plasma glass sheets"
+  id: Materialplasmaglass
```
Comment (Zumorica) [inline Resources/Prototypes/Catalog/Cargo/cargo_materials.yml:51]: This ID doesn't follow our [naming guidelines](https://hackmd.io/Awj5TvdXTaOmhU2XZOsdQg#Naming).

**Comment by metalgearsloth** [comment]
Vera didn't say to add plasma into cargo.

**Review by Zumorica — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Construction/Graphs/window.yml (hunk 1 above)
Comment (SethLafuente) [reply Resources/Prototypes/Recipes/Construction/Graphs/window.yml:14]: ok so to make plasma glass you need 1 plasma and 1 glass and to make plasma windows you need plasma not plasma glass just plasma which makes no sense whats the point of plasma glass if it dosen't work for making plasma windows 
image

Comment (Zumorica) [reply Resources/Prototypes/Recipes/Construction/Graphs/window.yml:14]: Your new recipe makes no sense, you clearly don't know what you're doing... If you're gonna make this use plasma glass sheets, *remove glass as a material in the recipe*

**Pull request closed without merging**